tox.ini

```
@@ -25,6 +25,15 @@ deps =
 commands =
     pylint energy_datetime
 
+[testenv:type_check]
+# the type_check environment checks the type hints using mypy
+deps =
+    -rrequirements.txt
+    mypy
+commands =
+    mypy --show-error-codes energy_datetime
```

remember to change this if #10 is merged first
```suggestion
    mypy --show-error-codes bdew_calendar
```